examples/dynamo_integration/README.md

```
@@ -46,45 +46,56 @@ source "${HOME}/.venvs/nat_dynamo_eval/bin/activate"
+#   - 4x NVIDIA GPUs (set WORKER_GPUS to the available set of machines)
+#   - Model weights: downloaded per previous instructions
+#   - Check that default ports are available: 8099 (HTTP API), 2389 (ETCD), 4232 (NATS)
+
+# 7. Run evaluation
 cd ../../ # NeMo-Agent-Toolkit/
 nat eval --config_file examples/dynamo_integration/react_benchmark_agent/configs/eval_config_no_rethinking_full_test.yml
 ```
 
+> [!WARNING]
+> The first load of model weights to `SGLang` workers can take significant time.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)